cache/save.go

```diff
@@ -7,17 +7,20 @@ import (
 	"strings"
 	"time"
 
-	"github.com/bitrise-io/go-steputils/v2/cache/compression"
-	"github.com/bitrise-io/go-steputils/v2/cache/keytemplate"
-	"github.com/bitrise-io/go-steputils/v2/cache/network"
-	"github.com/bitrise-io/go-steputils/v2/stepconf"
 	"github.com/bitrise-io/go-utils/v2/env"
 	"github.com/bitrise-io/go-utils/v2/log"
 	"github.com/bitrise-io/go-utils/v2/pathutil"
 	"github.com/bmatcuk/doublestar/v4"
 	"github.com/docker/go-units"
+
+	"github.com/bitrise-io/go-steputils/v2/cache/compression"
+	"github.com/bitrise-io/go-steputils/v2/cache/keytemplate"
+	"github.com/bitrise-io/go-steputils/v2/cache/network"
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
 )
 
+const MultipartFileSizeLimit = 200 * 1024 * 1024 // MultipartFileSizeLimit above 200MB we upload using multi-part
```

Comment:
https://github.com/bitrise-io/advanced-build-cache-service/pull/110/files#r1387447015

it looks like there is mismatch in cache service?

Reply:
Yeah, thanks for spotting. I think this is a leftover and can be removed. Initially, we wanted to call the new API only for multiparts, but we realized we can't do that as the backend needs to decide if the upload goes to GCS (which has no multipart support), so we moved all calls to the new API